File: week 7/task4.cpp
#include <iostream>
using namespace std;
void triangularNumber(int triangle);
main()
{
    int triangle;
    cout << "Enter the number of triangle: ";
    cin >> triangle;
    triangularNumber( triangle);

}
void triangularNumber(int triangle)
{
    int result = triangle * (triangle + 1) / 2;
    cout << "The number of dots is: " << result << endl;
}

File: lab manual 14/Task4.py
def find_length(num):
    count=0
    while(num!=0):
        num=int(num/10)
        count+=1
    return count    
num=int(input("Enter the length of number: "))
result=find_length(num)
print(f"The length of the {num} is: {result}")
